### PR TITLE
Close #LGVISIUM-94: Renaming of the output from create png api path

### PR DESCRIPTION
**Pull request by dcleres**
As the API is returning the key within the S3 bucket where the created PNG file is stored, it felt necessary to adapt the naming in the API response as well to avoid any type of confusion regarding what is returned.

### CODE REVIEW

**Comment by dcleres** [comment]
[@flenny ](https://github.com/flenny) do you think you can adapt the borehole application to cope with this future change?

File: src/app/common/schemas.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ class Config:
 class PNGResponse(BaseModel):
     """Response schema for the create_pngs endpoint."""
 
-    png_urls: list[str]
+    key: list[str]  # key in the S3 bucket
```
Comment (stijnvermeeren-swisstopo) [inline src/app/common/schemas.py:45]: Maybe `keys` (plural) would be better than `key` , since we have a list of values, not just a single value?

**Review by stijnvermeeren-swisstopo — CHANGES_REQUESTED** [review]

File: src/app/common/schemas.py (hunk 1 above)
Comment (dcleres) [reply src/app/common/schemas.py:45]: I think this is a good idea!

**Comment by dcleres** [comment]
@stijnvermeeren-swisstopo, thank you for your feedback. I made the required changes and re-ran the tests to see if everything was alright.

**Review by stijnvermeeren-swisstopo — APPROVED** [review]
LGTM!

**Pull request merged**